### ISSUE TITLE
Missing favicon.ico in develdocs.phpmyadmin.net

### ISSUE THREAD

**Issue by emanuelb** [phpmyadmin/website#43]
Visit:
https://develdocs.phpmyadmin.net/
Result (in console)
```
https://develdocs.phpmyadmin.net/favicon.ico Failed to load resource: the server responded with a status of 404 ()
```

**Comment by nijel** [comment]
Something can be added to https://github.com/phpmyadmin/scripts/tree/master/develdocs/output

**Comment by nijel** [comment]
Fixed in phpmyadmin/scripts@68eceeb180c29ad043eb415d87e25d909a0bb066
